astroquery/nasa_exoplanet_archive/core.py

```diff
@@ -246,9 +304,14 @@ def query_object_async(self, object_name, *, table="exoplanets", get_query_paylo
         References
         ----------
 
-        .. [1] `NASA Exoplanet Archive API Documentation
+        .. [1] `NASA Exoplanet Archive TAP Documentation
+           <https://exoplanetarchive.ipac.caltech.edu/docs/TAP/usingTAP.html>`_
+        .. [2] `NASA Exoplanet Archive API Documentation
            <https://exoplanetarchive.ipac.caltech.edu/docs/program_interfaces.html>`_
         """
+        # if table.lower() in ["ps"]: # actually want to check if default was used, but wasn't working ...
```

Comment:
It would be possible to use `table=None` and then show this warning, but this could also be annoying as a user. Someone who knows more about the astroquery ecosystem should comment on how best to handle deprecations like this.

Reply:
I'm pretty sure we have deprecation warning decorators.  But I'm not sure how important a warning this is.  Should users expect dramatically different behavior?

Reply:
Querying the PS table will generally return multiple rows per object, as opposed to only one per object for the old EXOPLANETS table, which is why we wanted the warning there if the new default is used. If specifically querying the old tables we want the code to break with an error message detailing the change in Archive 2.0, rather than just throwing a deprecation warning.

Reply:
@dfm, @keflavich - I've added a few mock tests for TAP. Let me know if you've had time to test out the functionality and are ready to merge this fix. The NExScI managers are getting impatient :P

Reply:
Was this comment meant for this specific line?  It looks like there is no warning here, right?